### PR TITLE
[ti_crowdstrike] Typecast the Cursor Value and Update Readme.

### PR DESCRIPTION
**Pull request by mohitjha-elastic**
Type of change
- BugFix

## What does this PR do?
**1. TypeCast the cursor value to string**
Typecast the cursor value to string as it is throwing error after intervals -
```
"error": {
      "message": "failed eval: no such overload"
    },
```

**2. Update the Readme**
Add user roles and management documentation link in readme.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### All changes
- [x] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md)
- [x] Supported versions of the monitoring target are documented
- [x] Supported operating systems are documented (if applicable)
- [x] Integration or [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) exist
- [x] Documentation exists
- [x] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [x] At least a manual test with ES / Kibana / Agent has been performed.
- [x] Required Kibana version set to: ^8.11.0

## How to test this PR locally
Clone integrations repo.
Install the elastic package locally.
Start the elastic stack using the elastic package.
Move to integrations/packages/ti_crowdstrike directory.
Run the following command to run tests.
`elastic-package test -v`

## Automated Test

[ti_crowdstrike-test-logs.txt](https://github.com/elastic/integrations/files/14374487/ti_crowdstrike-test-logs.txt)

### CODE REVIEW

File: packages/ti_crowdstrike/_dev/build/docs/README.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ This module has been tested against the **CrowdStrike Falcon Intelligence API Ve
 
 Follow the [documentation](https://www.crowdstrike.com/blog/tech-center/consume-ioc-and-threat-feeds/) for enabling the scopes from the CrowdStrike console.
 
+User should either have admin role or Detection Exception Manager role to access IOCs endpoint. Follow the [documentation](https://falcon.crowdstrike.com/documentation/page/f20650df/default-roles-reference) for managing user roles and permissions.
```
Comment (kcreddy) [inline packages/ti_crowdstrike/_dev/build/docs/README.md:58]:
```suggestion
User should either have `admin` role or `Detection Exception Manager` role to access IOCs endpoint. Follow the [documentation](https://falcon.crowdstrike.com/documentation/page/f20650df/default-roles-reference) for managing user roles and permissions.
```

File: packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs [hunk 2]
```diff
@@ -35,7 +35,7 @@ program: |
     !state.want_more ?
       request("GET", state.url + "/intel/combined/indicators/v1?offset=0&sort=_marker.asc&limit=" + string(state.batch_size) + '&filter=_marker:>"' + (
         has(state.cursor) && has(state.cursor.last_timestamp) && state.cursor.last_timestamp != null ?
-          state.cursor.last_timestamp + '"'
+          string(state.cursor.last_timestamp) + '"'
```
Comment (kcreddy) [inline packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs:38]: Similar change applies to `ioc` also, right?

Comment (mohitjha-elastic) [reply packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs:38]: Nope. In IOC, the field is already coming in `string` so no need to typecast that.
Tested on live instance.

Comment (kcreddy) [reply packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs:38]: Alright since you tested

**Review by kcreddy — APPROVED** [review]
LGTM

**Pull request merged**